app/views/exception_handler/exceptions/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="jumbotron border border-danger exceptionBox">
+	<div>
+	 	<%= t('exception.user_text') %>
+		<i class="fas fa-2x fa-bug"></i>
```

Comment:
That looks quite ugly: Maybe the icon should be moved upwards or use a red version of the mampf logo with a broken M. Maybe we need a little animal mascot (the mampfling? a squirrel?), that is crying.

Reply:
That sounds good. But I guess these graphical things are outside of my expertise.